.github/workflows/check-pr.yml

```diff
@@ -0,0 +1,17 @@
+name: Check Pull Request
+
+on:
+  pull_request:
+    types: [opened, labeled, unlabeled, synchronize]
```

Comment:
For what is `synchronize` (without ending d?) used?

Reply:
pushing on the PR